assets/templates/datasetLandingPage/nomis.tmpl

```diff
@@ -0,0 +1,245 @@
+         </div>
+      </dl>
+   </div>
+</div>
+<div class="wrapper adjust-font-size--18 line-height--32">
+  <div class="col-wrap">
+    <div class="col col--lg-two-thirds col--md-two-thirds margin-top--4 link-adjust">
+      <section>
+        <div class="margin-bottom--4"><span class="dataset-description">{{ .Metadata.Description | markdown }}</span></div>
+      </section>
+      <section>
+        {{$datasetID := .DatasetLandingPage.DatasetID}}{{$v := .DatasetLandingPage.Version}}
+        <h2 class="font-size--32 line-height--40 font-weight-700 margin-bottom--2">Get the data</h2>
+        <a href="{{.DatasetLandingPage.NomisReferenceURL}}" class="btn btn--primary btn--thick" role="button" rel="external"><b>Access this data on NOMIS <span class="icon icon-external--light-small"></span></b></a>
+        <p class="font-size--18 line-height--32">This dataset is provided on NOMIS.</p>
+      </section>
+         <section>
+           <div class="table-of-contents print--avoid-break border-bottom--iron-sm border-bottom--iron-md">
+             <h2 class="font-size--32 line-height--40 font-weight-700 table-of-contents__title font-weight-700">Supporting information: table of contents</h2>
+             <ol id="toc" class="table-of-contents__list">
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-dimensions" class="js-scroll">In this dataset</a>
+               </li>
+               {{if .DatasetLandingPage.LatestChanges}}
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-changes" class="js-scroll">What has changed in this edition</a>
+               </li>
+               {{end}}
+               {{if .DatasetLandingPage.QMIURL}}
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-qmi" class="js-scroll">Quality and methodology information</a>
+               </li>
+               {{end}}
+               {{if .DatasetLandingPage.Methodologies}}
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-methodology" class="js-scroll">Methodologies</a>
+               </li>
+               {{end}}
+               {{if .DatasetLandingPage.Citation}}
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-license" class="js-scroll">Usage information</a>
+               </li>
+               {{end}}
+               {{if .DatasetLandingPage.HasOlderVersions}}
+               <li class="line-height--32 table-of-contents__item margin-bottom--1 margin-left--1">
+                 <a href="#id-previous" class="js-scroll">Previous versions</a>
+               </li>
+               {{end}}
+             </ol>
+           </div>
+        </section>
+
```

Comment:
Are we missing keywords section as this is present in the prototype

Reply:
When refining the task was told by @janderson2 to not have the keywords section (something along the lines that users do not find it adds any value and bulks out the page)